extensions/arc/src/ui/dashboards/postgres/postgresParameters.ts

```diff
@@ -147,21 +147,20 @@ export abstract class PostgresParametersPage extends DashboardPage {
 							cancellable: false
 						},
 						async (_progress, _token): Promise<void> => {
+							let session: azdataExt.AzdataSession | undefined = undefined;
 							try {
 								this.parameterUpdates!.forEach((value, key) => {
 									engineSettings.push(`${key}="${value}"`);
 								});
-								const session = await this._postgresModel.controllerModel.acquireAzdataSession();
-								try {
-									this.saveParameterEdits(engineSettings.toString(), session);
-								} finally {
-									session.dispose();
-								}
+								session = await this._postgresModel.controllerModel.acquireAzdataSession();
```

Comment:
You shouldn't need to change the session like this - a try/finally without a catch will still throw